src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -1,958 +1,217 @@
-                        tabPanel.setAttribute('aria-labelledby', tabId);
-                        tabPanel.querySelector('.tab-title').className = 'hidden';
-
-                        if (hasClass(tabNavigation[j], 'active')) {
-                            tabPanel.className = 'block';
-                            tabNavigation[j].setAttribute('aria-selected', 'true');
-                            tabNavigation[j].removeAttribute('tabindex');
-                        } else {
-                            tabPanel.className = 'hidden';
-                            tabNavigation[j].removeAttribute('aria-selected');
-                            tabNavigation[j].setAttribute('tabindex', '-1');
-                        }
+                toggle.addEventListener('click', (e) => {
+                    e.preventDefault();
 
-                        tabNavigation[j].addEventListener('click', function(e) {
-                            var activeTab = e.target || e.srcElement;
-
-                            /* needed because when the tab contains HTML contents, user can click */
-                            /* on any of those elements instead of their parent '<button>' element */
-                            while (activeTab.tagName.toLowerCase() !== 'button') {
-                                activeTab = activeTab.parentNode;
-                            }
-
-                            /* get the full list of tabs through the parent of the active tab element */
-                            var tabNavigation = activeTab.parentNode.children;
-                            for (var k = 0; k < tabNavigation.length; k++) {
-                                var tabId = tabNavigation[k].getAttribute('data-tab-id');
-                                document.getElementById(tabId).className = 'hidden';
-                                removeClass(tabNavigation[k], 'active');
-                                tabNavigation[k].removeAttribute('aria-selected');
-                                tabNavigation[k].setAttribute('tabindex', '-1');
-                            }
-
-                            addClass(activeTab, 'active');
-                            activeTab.setAttribute('aria-selected', 'true');
-                            activeTab.removeAttribute('tabindex');
-                            var activeTabId = activeTab.getAttribute('data-tab-id');
-                            document.getElementById(activeTabId).className = 'block';
-                        });
+                    if ('' !== window.getSelection().toString()) {
+                        /* Don't do anything on text selection */
+                        return;
                     }
 
-                    tabGroups[i].setAttribute('data-processed', 'true');
-                }
-            },
-
-            createToggles: function() {
-                var toggles = document.querySelectorAll('.sf-toggle:not([data-processed=true])');
+                    let toggle = e.target || e.srcElement;
```

Comment:
`e.srcElement` is for old IE, so this can be removed from the code targetting only modern browsers supporting private class fields.